### ISSUE TITLE
Update ElasticCache default limits

### ISSUE THREAD

**Issue by sebasrp** [jantman/awslimitchecker#488]
## Bug Report

When reporting a bug in awslimitchecker, please provide all of the following information,
as well as any additional details that may be useful in reproducing or fixing
the issue:

### Version

``awslimitchecker 9.0.0``

### Installation Method
Installed through pipenv and executed through python directly

### Supporting Software Versions
```
➜ pipenv run python --version
Python 3.8.5

➜ pipenv --version
pipenv, version 2020.8.13

➜ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.15.6
BuildVersion:	19G2021
```

### Actual Output

```
➜ awslimitchecker --region ap-southeast-1 -P pd-staging --service ElastiCache --list-defaults
awslimitchecker 9.0.0@9.0.0 is AGPL-licensed free software; all users have a right to the full source code of this version. See <https://github.com/jantman/awslimitchecker>
WARNING:awslimitchecker.trustedadvisor:Cannot check TrustedAdvisor: AWS Premium Support Subscription is required to use this service.
ElastiCache/Nodes                     100
ElastiCache/Nodes per Cluster         20
ElastiCache/Parameter Groups          20
ElastiCache/Security Groups           50
ElastiCache/Subnet Groups             50
ElastiCache/Subnets per subnet group  20
```

### Expected Output

Based on https://docs.aws.amazon.com/general/latest/gr/elasticache-service.html, these limits are no longer accurate and should be:
Limit | Current Value | New value
------------ | ------------- | -------------
ElastiCache/Nodes |  100 | 300
ElastiCache/Nodes per Cluster | 20 | ~~40~~ 20
ElastiCache/Parameter Groups |  20  | 150
ElastiCache/Security Groups | 50 | 
ElastiCache/Subnet Groups |  50 | 150
ElastiCache/Subnets per subnet group | 20

### TrustedAdvisor

Not Applicable

### Testing Assistance

Yes, i will be able to assist in testing pre-release code for the feature.

**Comment by sebasrp** [comment]
I am preparing a pull request for this update

**Comment by FabianSawatzki1234** [comment]
We ran into the ElastiCache limits as well and it's a false positve. Please merge the pull request.

**Comment by jantman** [comment]
Replied to the PR with a question regarding default limits. I'll get it merged this week. Sorry for the delay.

**Comment by sebasrp** [comment]
Seems like the limits I specified in the description should be these updates instead

Limit | Current Value | New value
------------ | ------------- | -------------
ElastiCache/Nodes |  100 | 300
ElastiCache/Nodes per Cluster | 20 | ~~40~~ 20
ElastiCache/Parameter Groups |  20  | 150
ElastiCache/Security Groups | 50 | 
ElastiCache/Subnet Groups |  50 | 150
ElastiCache/Subnets per subnet group | 20

**Comment by sebasrp** [comment]
updated PR to match this.

**Comment by jantman** [comment]
A fix for this issue has been merged to the [develop](https://github.com/jantman/awslimitchecker/tree/develop) branch and is slated for release in 10.0.0 on Monday, December 7, 2020. Anyone who is able to test the fix is strongly encouraged to do so, and comment on this issue with their results. Instructions for installing awslimitchecker from a git branch are available in the [Installing for Development documentation](https://awslimitchecker.readthedocs.io/en/latest/development.html#installing-for-development).

**Comment by jantman** [comment]
This has been fixed in [10.0.0](https://github.com/jantman/awslimitchecker/releases/tag/10.0.0), which is now [live on PyPI](https://pypi.org/project/awslimitchecker/) and is being built on the [Docker hub](https://hub.docker.com/r/jantman/awslimitchecker) right now. Thank you so much for the contribution!
